forge-gui/res/cardsfolder/upcoming/bind_to_secrecy.txt

```diff
@@ -0,0 +1,18 @@
+Name:Bind to Secrecy
+ManaCost:U B
+Types:Instant
+A:SP$ Charm | Choices$ CounterNonCreature,DBConjure | CharmNum$ 1 | SpellDescription$ If there are five or more mana values among cards in your graveyard, draft a card from CARDNAME's spellbook.
+SVar:CounterNonCreature:DB$ Counter | TargetType$ Spell | TgtPrompt$ Select target noncreature spell | ValidTgts$ Card.nonCreature | SubAbility$ TrigDraft | SpellDescription$ Counter target noncreature spell.
+SVar:DBConjure:DB$ MakeCard | TgtPrompt$ Select target creature card in an opponent's graveyard | ValidTgts$ Creature.OppOwn | TgtZone$ Graveyard | DefinedName$ Targeted | Zone$ Hand | RememberMade$ True | SubAbility$ DBEffect | SubAbility$ TrigDraft | SpellDescription$ Conjure a duplicate of target creature card in an opponent's graveyard into your hand. It perpetually gains "You may spend mana as though it were mana of any color to cast this spell."
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | StaticAbilities$ PerpetualAbility | Duration$ Permanent | Name$ Bind to Secrecy's Perpetual Effect | Triggers$ Update | SubAbility$ DBCleanup
```

Comment:
A bug here slipped past me in scripting. The `DBEffect` is not correctly applying the perpetual cast-with-any-color effect to the conjured card. Working on a fix now

Reply:
Resolved